src/main/java/org/eclipse/tractusx/ssi/lib/proof/LinkedDataProofGenerator.java

```diff
@@ -59,9 +62,11 @@ public static LinkedDataProofGenerator newInstance(SignatureType type)
     if (type == SignatureType.ED25519) {
       return new LinkedDataProofGenerator(
           type, new LinkedDataHasher(), new LinkedDataTransformer(), new Ed25519ProofSigner());
-    } else {
+    } else if (type == SignatureType.JWS) {
```

Comment:
this was  a else branch without if on purpose, since there are other SignatureTypes like:
* JWS_P256
* JWS_RSA
* ...

Reply:
Should we take our(Cofinity-X) changes here?

Reply:
Fixed with https://github.com/Cofinity-X/SSI-agent-lib/pull/49/commits/54b99a4ca55d4deb7c3d58799b8de51bc33c7e01